ngraph/frontend/onnx/frontend/src/op/qlinear_conv.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// Disabled in CMakeList
+// Update to higher opset required
+
+#include "op/qlinear_conv.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "conv.hpp"
+#include "dequantize_linear.hpp"
+#include "exceptions.hpp"
+#include "ngraph/opsets/opset6.hpp"
+#include "onnx_import/core/null_node.hpp"
+#include "quantize_linear.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector qlinear_conv(const Node& node) {
+    const OutputVector& inputs = node.get_ng_inputs();
+
+    auto x = inputs.at(0);
+    auto x_scale = inputs.at(1);
+    auto x_zero_point = inputs.at(2);
+    auto w = inputs.at(3);
+    auto w_scale = inputs.at(4);
+    auto w_zero_point = inputs.at(5);
+    auto y_scale = inputs.at(6);
+    auto y_zero_point = inputs.at(7);
+    Output<ngraph::Node> B = inputs.size() > 8 ? inputs.at(8) : std::make_shared<NullNode>()->output(0);
```

Comment:
I think you can simplify this logic and get rid of `NullNode` and `op::is_null`, just checking the input sizes instead.